Content.Server/_DV/Curation/Systems/CwoinkSystem.cs

```diff
@@ -0,0 +1,1045 @@
+        var roundTime = _gameTicker.RoundDuration().ToString("hh\\:mm\\:ss");
+
+        // Determine the icon based on the status type
+        string icon = statusType switch
+        {
+            PlayerStatusType.Connected => ":green_circle:",
+            PlayerStatusType.Disconnected => ":red_circle:",
+            PlayerStatusType.Banned => ":red_circle:",
+            _ => ":question:"
+        };
+
+        // Create the message parameters for Discord
+        var messageParams = new CHelpMessageParams(
+            session.Name,
+            message,
+            true,
+            roundTime,
+            _gameTicker.RunLevel,
+            playedSound: true,
+            icon: icon
+        );
+
+        // Create the message for in-game with username
+        var color = statusType switch
+        {
+            PlayerStatusType.Connected => Color.Green.ToHex(),
+            PlayerStatusType.Disconnected => Color.Yellow.ToHex(),
+            PlayerStatusType.Banned => Color.Yellow.ToHex(),
+            _ => Color.Gray.ToHex(),
+        };
+        var inGameMessage = $"[color={color}]{session.Name} {message}[/color]";
+
+        var cwoinkMessage = new CwoinkTextMessage(
+            userId: session.UserId,
+            trueSender: SystemUserId,
+            text: inGameMessage,
+            sentAt: DateTime.Now,
+            playSound: false
+        );
+
+        var admins = _activeCurators;
+        foreach (var admin in admins)
+        {
+            RaiseNetworkEvent(cwoinkMessage, admin);
+        }
+
+        // Enqueue the message for Discord relay
+        if (_webhookUrl != string.Empty)
+        {
+            // if (!_messageQueues.ContainsKey(session.UserId))
+            //     _messageQueues[session.UserId] = new Queue<string>();
+            //
+            // var escapedText = FormattedMessage.EscapeText(message);
+            // messageParams.Message = escapedText;
+            //
+            // var discordMessage = GenerateAHelpMessage(messageParams);
+            // _messageQueues[session.UserId].Enqueue(discordMessage);
+
+            var queue = _messageQueues.GetOrNew(session.UserId);
+            var escapedText = FormattedMessage.EscapeText(message);
+            messageParams.Message = escapedText;
+            var discordMessage = GenerateAHelpMessage(messageParams, _discordReplyPrefix);
+            queue.Enqueue(discordMessage);
+        }
+    }
+
+    private void OnGameRunLevelChanged(GameRunLevelChangedEvent args)
+    {
+        // Don't make a new embed if we
+        // 1. were in the lobby just now, and
+        // 2. are not entering the lobby or directly into a new round.
+        if (args.Old is GameRunLevel.PreRoundLobby ||
+            args.New is not (GameRunLevel.PreRoundLobby or GameRunLevel.InRound))
+        {
+            return;
+        }
+
+        // Store the Discord message IDs of the previous round
+        _oldMessageIds = new Dictionary<NetUserId, string>();
```

Comment:
No need to make a new dict
```suggestion
        _oldMessageIds.Clear();
```